blog/2022-06-30-luos-and-esp32.mdx

```diff
@@ -0,0 +1,179 @@
+---
+title: "Luos and ESP32"
+description: Luos engine is plug-and-play compatible with the ESP32 family! Luos and Espressif have many concepts in common, find out more in this article.
+slug: luos-and-esp32
+authors:
+  - name: Thomas Salembier
+    title: CTO at Luos
+    url: https://github.com/Salem-Luos
+    image_url: /img/school/authors/thomas.png
+tags: [esp32, iot, embedded, edge, systems]
+image: /img/blog/ESP32-and-Luos.png
+hide_table_of_contents: false
+date: 2022-06-30T11:00
+---
+
+![Luos and ESP32](/img/blog/ESP32-and-Luos.png)
+
+With the Luos development team, we are constantly working on integrating our solution with the tools you use. 
+
+Great News! Luos engine is now plug-and-play-compatible with the ESP32 family!
+
+<!--truncate-->
+
+<h2>What is ESP32?</h2>
+
+The ESP32 is a low-cost, low-power microcontroller with built-in Wi-Fi and Bluetooth Classic/BLE support.
+
+The ESP32 is a successor to the ESP8266, a breakout hit on Kickstarter. The ESP32 adds Wi-Fi and Bluetooth Classic/BLE support to the already impressive list of features of the ESP8266.
+
+ESP32 is created by Espressif Systems, a Shanghai-based company with an excellent reputation for producing low-cost, low-power microcontrollers.
+
+ESP32 is based on Tensilica's Xtensa LX6 architecture, incorporating dual-mode Wi-Fi, Bluetooth management, and a DSP. It is often dedicated to the Internet of Things (IoT) and embedded applications because of its low price.
+
+The small ESP32 package has a high level of integrations, such as:
+
+- RF balun
+- In-built antenna switches
+- Power amplifier
+- Low-noise receive amplifier
+- Filters
+- Power management modules
+- With a Printed Circuit Board (PCB), ESP32 provides an accessible and versatile application for your projects.
+
+Indeed, in the new Luos' release 2.5.0, a Hardware Abstraction Layer (HAL) of Luos engine and an example will allow you to test the Luos technology on those MCU and develop your services for ESP32 family. Follow our tutorial to start with Luos and ESP32.
+
+Luos team developed a HAL based on ESP-IDF SDK provided by Espressif to create a Robus network and natively exchange messages and information through the Luos protocol.
+
+This new covering of MCU family makes Luos compatible with the IoT world product development. It offers the possibility to make new tools for Luos like a gate and an inspector to communicate with an embedded product on a computer through Wi-Fi or Bluetooth.
```

Comment:
"Luos like a gate and an inspector" a link to the documenation may be good

Reply:
Resolved